tests/setup-travis.sh

```diff
@@ -49,7 +49,10 @@ if [ $clone_exit_code -ne 0 ]; then
 fi
 
 cd ..
+# Plugin dir for WP <= 5.5.1
 cp -r $PLUGIN_SLUG "/tmp/wordpress-$WP_BRANCH/src/wp-content/plugins/$PLUGIN_SLUG"
+# Plugin dir for WP > 5.5.1
```

Comment:
This would likely be more accurate to say >= 5.6-alpha. There can be releases on the 5.5 branch after 5.6 is out (security and the like).